.gitignore

```diff
@@ -131,5 +131,6 @@ dmypy.json
 # pdm
 .pdm.toml
 
-# vs-code user settings
-.vscode
+# VSCode
+.vscode/**/*
+!.vscode/settings.json

```

Comment:
Are you sure this is the best way to share those settings? I tend to change stuff in this file and having it tracked by git may not be comfortable.

How about storing it in a separate folder where we can copy it from?

Reply:
I'm certain that this is not the best way to actually share those settings. I will try to create a default file that you can copy to create your own settings.json file

Reply:
Looks good to me.